components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -101,6 +118,9 @@
 typedef NS_ENUM(NSInteger, MDCTypographySchemeDefaults) {
   /**
    The Material defaults, circa April 2018.
+
+   MDCTypographyScheming's useCurrentContentSizeCategoryWhenApplied is assigned to NO by default
```

Comment:
_Tip:_
```diff
-  MDCTypographyScheming's useCurrentContentSizeCategoryWhenApplied is assigned to NO by default with this version.
+ @c useCurrentContentSizeCategoryWhenApplied defaults to @c NO.
```

Reply:
I think the latter is a bit less clear if someone doesn't know what a `useCurrentContentSizeCategoryWhenApplied` property is or where it's configured.